techniques/applications/systemUpdateCampaign/1.0/metadata.xml

```diff
@@ -0,0 +1,53 @@
+<TECHNIQUE name="System Update Campaign">
+  <DESCRIPTION>plouf</DESCRIPTION>
+  <MULTIINSTANCE>false</MULTIINSTANCE>
+  <POLICYGENERATION>separated</POLICYGENERATION>
+  <USEMETHODREPORTING>true</USEMETHODREPORTING>
+
+  <BUNDLES>
+    <NAME>rudder_system_update_campaign</NAME>
+  </BUNDLES>
+
+  <TMLS>
+    <TML name="systemUpdate"/>
+  </TMLS>
+
+  <SYSTEMVARS>
+    <NAME>AGENT_RUN_INTERVAL</NAME>
+  </SYSTEMVARS>
+
+  <SECTIONS>
+    <SECTION name="Update the system" reporting="worst-case-weighted-one" id="Update the system" component="true">
+      <SECTION name="${update}" id="system-update" multivalued="true" component="true">
+        <REPORTKEYS>
+          <VALUE id="system-update">${updateValue}</VALUE>
+        </REPORTKEYS>
+      </SECTION>
+    </SECTION>
+    <SECTION name="Campaign Infos" multivalued="false" component="false">
+      <INPUT>
+        <NAME>ENABLE_UPDATE</NAME>
+        <DESCRIPTION>Activate or not the update. Must be "true", "false" or "enabled", "disabled"</DESCRIPTION>
```

Comment:
The description does not match the class definitions below.
The cf code only expects `enable` here.